src/Microsoft.Extensions.Options/OptionsServiceCollectionExtensions.cs

```diff
@@ -37,7 +39,18 @@ public static IServiceCollection AddOptions(this IServiceCollection services)
         /// <param name="services">The <see cref="IServiceCollection"/> to add the services to.</param>
         /// <param name="configureOptions">The action used to configure the options.</param>
         /// <returns>The <see cref="IServiceCollection"/> so that additional calls can be chained.</returns>
-        public static IServiceCollection Configure<TOptions>(this IServiceCollection services, Action<TOptions> configureOptions)
+        public static IServiceCollection Configure<TOptions>(this IServiceCollection services, Action<TOptions> configureOptions) where TOptions : class
+            => services.Configure(string.Empty, configureOptions);
```

Comment:
Also, we also have the option of making legacy options to configure ALL instances.